Android builds fails with clean flutter 2.2 project

Simply including karte_core in to pubspec.yaml causes the build to fail.

### Steps to reproduce
1. flutter create myproject
2. Add karte_core: 0.1.0 to dependencies of pubspec.yaml
3. flutter build apk

### Build error
```
e: /xxxxxxx/source/flutter-2.2/.pub-cache/hosted/pub.dartlang.org/karte_core-0.1.0/android/src/main/kotlin/io/karte/flutter/core/KarteCorePlugin.kt: (135, 48): Unresolved reference: VERSION_NAME
                                                                        
FAILURE: Build failed with an exception.
```

The reason for this is Flutter has updated to gradle 4+, which does not expose the VERSION_NAME to plugins.

--- a/karte_core/android/build.gradle
+++ b/karte_core/android/build.gradle
@@ -37,6 +37,7 @@ android {
     defaultConfig {
         minSdkVersion 16
         versionName version
+        buildConfigField "String", "LIB_VERSION", "\"$version\""
     }
     lintOptions {
         disable 'InvalidPackage'

--- a/karte_core/android/src/main/kotlin/io/karte/flutter/core/KarteCorePlugin.kt
+++ b/karte_core/android/src/main/kotlin/io/karte/flutter/core/KarteCorePlugin.kt
@@ -132,7 +132,7 @@ class KarteCorePlugin : FlutterPlugin, MethodCallHandler, Library {
     //region Library
     override val isPublic: Boolean = true
     override val name: String = "flutter"
-    override val version: String = BuildConfig.VERSION_NAME
+    override val version: String = BuildConfig.LIB_VERSION
     override fun configure(app: KarteApp) {}
     override fun unconfigure(app: KarteApp) {}
     //endregion

--- a/karte_core/pubspec.yaml
+++ b/karte_core/pubspec.yaml
@@ -1,6 +1,6 @@
 name: karte_core
 description: Flutter plugin for KARTE Core.
-version: 0.1.0
+version: 0.1.1
 homepage: https://github.com/plaidev/karte-flutter/tree/master/karte_core
 
 environment:

